modules/spo-recording-profiles.adoc

```diff
@@ -95,36 +97,68 @@ my-pod   2/2     Running   0          18s
 ----
 $ oc -n openshift-security-profiles logs --since=1m --selector name=spod -c log-enricher
 ----
+
+ifdef::seccomp[]
++
+.Example output
+[source,terminal]
+----
+I0523 14:19:08.747313  430694 enricher.go:445] log-enricher "msg"="audit" "container"="redis" "executable"="/usr/local/bin/redis-server" "namespace"="my-namespace" "node"="xiyuan-23-5g2q9-worker-eastus2-6rpgf" "pid"=656802 "pod"="my-pod" "syscallID"=0 "syscallName"="read" "timestamp"="1684851548.745:207179" "type"="seccomp"
+----
+endif::[]
+
+ifdef::selinux[]
 +
 .Example output
 [source,terminal,subs="attributes+"]
 ----
-I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "perm"="name_bind" "pod"="my-pod" "profile"="test-recording_redis_6kmrb_1684331729" "scontext"="system_u:system_r:selinuxrecording.process:s0:c4,c27" "tclass"="tcp_socket" "tcontext"="system_u:object_r:redis_port_t:s0" "timestamp"="1684331735.105:273965" "type"="{type}"
+I0517 13:55:36.383187  348295 enricher.go:376] log-enricher "msg"="audit" "container"="redis" "namespace"="my-namespace" "node"="ip-10-0-189-53.us-east-2.compute.internal" "perm"="name_bind" "pod"="my-pod" "profile"="test-recording_redis_6kmrb_1684331729" "scontext"="system_u:system_r:selinuxrecording.process:s0:c4,c27" "tclass"="tcp_socket" "tcontext"="system_u:object_r:redis_port_t:s0" "timestamp"="1684331735.105:273965" "type"="selinux"
 ----
+endif::[]
+
 
 .Verification
 
 . Remove the pod:
 +
 [source,terminal]
 ----
-$ oc -n openshift-security-profiles delete pod my-pod
+$ oc -n my-namepace delete pod my-pod
 ----
 
 . Confirm the Security Profiles Operator reconciles the two {type} profiles:
+
+ifdef::seccomp[]
 +
-[source,terminal,subs="attributes+"]
+[source,terminal]
 ----
-$ oc get {object} -n my-namespace
+$ oc get seccompprofiles -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
 ----
 +
-.Example output
+.Example output for seccompprofile
+[source,terminal]
+----
+NAME                   STATUS      AGE
+test-recording-nginx   Installed   2m48s
+test-recording-redis   Installed   2m48s
+----
+endif::[]
+
+ifdef::selinux[]
++
+[source,terminal]
+----
+$ oc get selinuxprofiles -lspo.x-k8s.io/recording-id=test-recording -n my-namespace
+----
++
+.Example output for selinuxprofile
 [source,terminal]
 ----
-NAME                   USAGE                                       STATE
+NAME                   USAGE                                       State
```

Comment:
Need to change it back to STATE.
NOTE: Only the headers needed to be updated. Below is just an example to show what the headers should be.
```
$ oc get selinuxprofiles
NAME          USAGE                                            STATE
errorlogger   errorlogger_security-profiles-operator.process   Installed
```